rest/taskrouter/twiml/example3/example/example.3.x.js

```diff
@@ -0,0 +1,28 @@
+// Download the Node helper library from twilio.com/docs/node/install
+// These consts are your accountSid and authToken from https://www.twilio.com/console
+const twilio = require('twilio');
+const express = require('express');
+const app = express();
+
+app.post('/enqueue_call', function(request, response) {
+    const resp = new twilio.TwimlResponse();
+
+    const arr = {account_number: '12345abcdef'};
+    const json = JSON.stringify(arr);
+
+    resp.enqueue(
+      {workflowSid: 'WW0123456789abcdef0123456789abcdef'},
+      (node) => {
+        node.task(json, {
```

Comment:
Just my opinion, but you could move this anonymous function and define it above for readability. I can read it either way though. +1 for readability throughout!